### PR TITLE
pio-team: add page

### PR DESCRIPTION
**Pull request by marchersimon**
Part of #5517

### CODE REVIEW

File: pages/common/pio-team.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add a new member to a team:
+
+`pio team add {{orgname}}:{{teamname}} {{username}}`
+
+- Remove a member from a team:
+
+`pio team remove {{orgname}}:{{teamname}} {{username}}`
+
+- List all teams and their members, the user is part of:
```
Comment (bl-ue) [inline pages/common/pio-team.md:22]:
```suggestion
- List all teams and all their members:
```

File: pages/common/pio-team.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Delete a team:
+
+`pio team destroy {{orgname}}:{{teamname}}`
+
+- Add a new member to a team:
+
+`pio team add {{orgname}}:{{teamname}} {{username}}`
+
+- Remove a member from a team:
```
Comment (bl-ue) [inline pages/common/pio-team.md:18]:
```suggestion
- Remove a user from a team:
```

File: pages/common/pio-team.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Create a new team with a description:
+
+`pio team create --description {{description}} {{orgname}}:{{teamname}}`
+
+- Delete a team:
+
+`pio team destroy {{orgname}}:{{teamname}}`
+
+- Add a new member to a team:
```
Comment (bl-ue) [inline pages/common/pio-team.md:14]:
```suggestion
- Add a new user to a team:
```

File: pages/common/pio-team.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Remove a member from a team:
+
+`pio team remove {{orgname}}:{{teamname}} {{username}}`
+
+- List all teams and their members, the user is part of:
+
+`pio team list`
+
+- List all teams and their members of an organisation:
```
Comment (bl-ue) [inline pages/common/pio-team.md:26]:
```suggestion
- List all members of an organization:
```

File: pages/common/pio-team.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`pio team list {{orgname}}`
+
+- Rename a team and update its description:
+
+`pio team update --name {{new_teamname}} --description {{new_description}} {{orgname}}:{{teamname}}`
```
Comment (bl-ue) [inline pages/common/pio-team.md:32]: As there's room, I'd split this into two examples.
```suggestion
- Rename a team:

`pio team update --name {{new_teamname}} {{orgname}}:{{teamname}}`

- Change the description of a team:

`pio team update --description {{new_description}} {{orgname}}:{{teamname}}`
```

File: pages/common/pio-team.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/pio-team.md:22]: Don't you think `all teams` could be confused with "all teams there are"?

File: pages/common/pio-team.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/pio-team.md:26]: Isn't that an oversimplification? Your suggeston would be `pio org list`, but here it also lists all teams.

File: pages/common/pio-team.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/pio-team.md:22]: I intended it to mean that—is it not true? https://docs.platformio.org/en/latest/core/userguide/team/cmd_list.html seems to me to mean that.

File: pages/common/pio-team.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/pio-team.md:26]: Updated.  Does that sound better?

File: pages/common/pio-team.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/pio-team.md:22]: Like in `pio org list`, when I run `pio team list`, I get `You do not have any teams`.

File: pages/common/pio-team.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`pio team list`
+
+- List all teams in an organization:
+
+`pio team list {{organization_name}}`
+
+- Rename a team:
+
+`pio team update --name {{new_teamname}} {{organization_name}}:{{team_name}}`
```
Comment (navarroaxel) [inline pages/common/pio-team.md:32]:
```suggestion
`pio team update --name {{new_team_name}} {{organization_name}}:{{team_name}}`
```

File: pages/common/pio-team.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# pio team
+
+> Manage PlatformIO teams.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/team/>.
+
+- Create a new team with the specified description:
+
+`pio team create --description {{description}} {{orgname}}:{{team_name}}`
```
Comment (navarroaxel) [inline pages/common/pio-team.md:8]:
```suggestion
`pio team create --description {{description}} {{organization_name}}:{{team_name}}`
```

File: pages/common/pio-team.md (hunk 1 above)
Comment (sbrl) [reply pages/common/pio-team.md:22]: What about this?
```suggestion
- List all teams that the user is part of and their members:
```

This seems to me like a word-ordering issue

**Review by navarroaxel — APPROVED** [review]

**Review by yutyo — APPROVED** [review]
LGTM.

**Review by bl-ue — APPROVED** [review]

**Pull request merged**